app/models/goal.py

```diff
@@ -1,5 +1,27 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import Integer, String
 from ..db import db
 
 class Goal(db.Model):
-    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    __tablename__ = "goals"
+
+    id: Mapped[int]    = mapped_column(Integer, primary_key=True, autoincrement=True)
+    title: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
In the current version of SQLAlchemy `nullable=False` is the default so we should leave off `mapped` with `nullable` because this adds more code to the codebase that isn't needed.

```suggestion
    title: Mapped[str] 
```